CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ DEPRECATIONS/CHANGES:
 
  * Step-Down is Forwarded: When a step-down is issued against a non-active node
    in an HA cluster, it will now forward the request to the active node.
+ * HMAC Keys in Transit Use Min Encryption Version: In prior releases, the
```

Comment:
Huh. not sure why I thought this was different. Yeah, it would always use the latest version, like the other keys.